apps/typegpu-docs/astro.config.mjs

```diff
@@ -76,13 +84,22 @@ export default defineConfig({
             //   label: 'Parametrized Functions',
             //   slug: 'guides/parametrized-functions',
             // },
+          ]),
+        },
+        DEV && {
+          label: 'Tutorials',
+          items: [
+            {
+              label: 'Making Triangles Fly',
```

Comment:
A proposal for an alternate name for the tutorial 👀. Thoughts?

Reply:
it's cool, though I personally prefer the old one. and I think the name in the menu should match the guide's heading

Reply:
Maybe we could merge them? 
'Triangle to boids - making triangles fly'

Reply:
![image](https://github.com/user-attachments/assets/aa8f23ad-32c8-4563-956f-8b04ae4ce3c9)